tests/integration/slashing.go

```diff
@@ -72,32 +72,31 @@ func (s *CCVTestSuite) TestRelayAndApplyDowntimePacket() {
 		timeoutHeight    = clienttypes.Height{}
 		timeoutTimestamp = uint64(s.getFirstBundle().GetCtx().BlockTime().Add(ccv.DefaultCCVTimeoutPeriod).UnixNano())
 	)
-	slashPacket := s.constructSlashPacketFromConsumer(s.getFirstBundle(), *tmVal, stakingtypes.Infraction_INFRACTION_DOWNTIME)
-	sequence, err := s.getFirstBundle().Path.EndpointA.SendPacket(timeoutHeight, timeoutTimestamp, slashPacket.GetBytes())
+	slashPacket := s.constructSlashPacketFromConsumer(s.getFirstBundle(), *tmVal, stakingtypes.Infraction_INFRACTION_DOWNTIME, 1)
+	sequence, err := s.getFirstBundle().Path.EndpointA.SendPacket(timeoutHeight, timeoutTimestamp, slashPacket.GetData())
 	s.Require().NoError(err)
 
 	// Set outstanding slashing flag for first consumer, it's important to use the consumer's cons addr here
 	firstConsumerKeeper.SetOutstandingDowntime(s.consumerCtx(), consumerConsAddr.ToSdkConsAddr())
 
 	// Note: RecvPacket advances two blocks. Let's say the provider is currently at height N.
-	// The received slash packet will be queued during N, and handled by the ccv module during
-	// the endblocker of N. The staking module will then register a validator update from that
-	// packet during the endblocker of N+1 (note that staking endblocker runs before ccv endblocker,
-	// hence why the VSC is registered on N+1). Then the ccv module sends VSC packets to each consumer
-	// during the endblocker of N+1. The new validator set will be committed to in block N+2,
-	// and will be in effect for the provider during block N+3.
+	// The received slash packet will be handled during N. The staking module will then register
+	// a validator update from that packet during the endblocker of N. Then the ccv module sends
+	// VSC packets to each consumer during the endblocker of N (note ccv endblocker runs after staking).
+	// The new validator set will be committed to in block N+1, and will be in effect
+	// for the provider during block N+2.
 
 	valsetUpdateIdN := providerKeeper.GetValidatorSetUpdateId(s.providerCtx())
 
 	// receive the slash packet on the provider chain. RecvPacket() calls the provider endblocker twice
```

Comment:
maybe we can check that explicitly? i.e 

`heightBefore := s.providerChain.currentHeader.Height` and then compare the height with the one after?

feel free to ignore this, think it's not crucial

Reply:
This is debatedly "testing the testing framework" :) but I see no problem quickly adding the assertion https://github.com/cosmos/interchain-security/commit/3cb1f47d536b13c4e4384c4972445d241056cd2c